packages/visual-dev/src/components/RadioCard/Styles.ts

```diff
@@ -4,7 +4,7 @@ export const RadioLabelStyle = css`
   display: flex;
   flex-direction: row;
   //   width: 85%;
-  min-height: 138px;
+  min-height: 110px;
```

Comment:
Should do a quick regression test to make sure this doesn't break other cards in the portal